Sources/Comet/CometClient/CometClient.swift

```diff
@@ -16,23 +16,31 @@ public final class CometClient {
     private let authenticator: Authenticator
     private let authenticatedRequestBuilder: AuthenticatedRequestBuilding
     private let requestResponseHandler: RequestResponseHandling
+    private let logLevel: RequestLogLevel
+    private let logger: (String) -> Void
 
     /// TODO
     /// - Parameters:
     ///   - urlSession: TODO
     ///   - tokenProvider: TODO
     ///   - authenticatedRequestBuilder: TODO
     ///   - requestResponseHandler: TODO
+    ///   - logLevel: TODO
+    ///   - logger: TODO
     public init(
         urlSession: URLSession = .shared,
         tokenProvider: TokenProviding,
         authenticatedRequestBuilder: AuthenticatedRequestBuilding,
-        requestResponseHandler: RequestResponseHandling = CometRequestResponseHandler()
+        requestResponseHandler: RequestResponseHandling = CometRequestResponseHandler(),
+        logLevel: RequestLogLevel = .full,
```

Comment:
I would probably give as the default value .none

Reply:
Agree. Added default value as `.none`